recipes-wpe/wpeframework/include/wpeframework-plugins-rdk.inc

```diff
@@ -28,7 +28,7 @@ PACKAGECONFIG ??= "\
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_vg', 'opencdmi_vgrdm', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'widevine', 'opencdmi_wv', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'security', 'securityagent', '', d)} \
-    deviceidentification deviceinfo locationsync monitor monitor_webkit tracing webkitbrowser \
+    deviceidentification deviceinfo locationsync monitor monitor_webkit messagecontrol webkitbrowser \
```

Comment:
Tracing module is still used by the integration. Please ensure that recipe is backward compatible

Reply:
We can use either message control or tracing. I will add PACKAGE CONFIG to enable message control by default and tracing if message control disabled case

Reply:
Sounds like a plan. Please do so.

Reply:
done